npm/angular/package.json

```diff
@@ -1,95 +1,67 @@
 {
   "name": "@cypress/angular",
   "version": "0.0.0-development",
-  "description": "Test Angular components using Cypress",
-  "private": true,
+  "description": "Test Angular Components using Cypress",
   "main": "dist/index.js",
   "scripts": {
-    "build": "tsc --project tsconfig.lib.json",
-    "watch": "tsc --project tsconfig.lib.json -w",
-    "cy:open": "node ../../scripts/cypress.js open --component",
-    "cy:run": "node ../../scripts/cypress.js run --component",
-    "ng": "ng",
-    "app-start": "ng serve",
-    "app-build": "ng build",
-    "test": "yarn cy:run",
-    "test-ci": "echo not testing in CI because process never exits",
-    "postinstall": "patch-package"
-  },
-  "dependencies": {
-    "@cypress/mount-utils": "0.0.0-development",
-    "debug": "^4.3.2"
+    "prebuild": "rimraf dist",
+    "build": "tsc || echo 'built, with type errors'",
```

Comment:
Realized that rollup is overkill for this package. The version of Angular we support expect `esm` packages so we only need to publish one entrypoint, so using `tsc` directly is sufficient.